docs/user-guide.md

```diff
@@ -0,0 +1,72 @@
+# User Guide
+
+## Features
+
+### How to verify a kani harness using the extension
+
+
+As soon as the rust file containing the proofs is opened using the Kani extension in a VSCode instance, you should see the Kani proofs loaded as regular unit tests in the Testing Panel on the left border of VSCode. This is how the testing page looks like when you click on the panel.
```

Comment:
Capitalize and use full names for the things you're talking about (i.e., use either "Kani VSCode Extension" or just "the extension", but not a mix).

Reply:
One sentence per line please.